Frends.PowerShell.RunScript/Frends.PowerShell.RunScript.Tests/UnitTests.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NUnit.Framework;
+
+namespace Frends.PowerShell.RunScript.Tests
+{
+    [TestFixture]
+    public class UnitTests
+    {
+        /// <summary>
+        /// Setup initializes PowerShell ExecutionPolicy for CurrentUser as Unrestricted so the task can run running scripts.
+        /// </summary>
+        [OneTimeSetUp]
+        public void Setup()
+        {
+            var command = "Set-ExecutionPolicy";
+            PowerShellParameter[] parameters = new PowerShellParameter[]
+            {
+                new PowerShellParameter
+                {
+                    Name = "ExecutionPolicy",
+                    Value = "Unrestricted"
+                },
+                new PowerShellParameter
+                {
+                    Name = "Scope",
+                    Value = "CurrentUser"
+                }
+            };
+            _ = PowerShell.RunCommand(command, parameters, new RunOptions());
```

Comment:
You don't need the `_ = ` here, since only calling this method will omit the return.
So you can change this line to just calling the method.
```suggestion
            PowerShell.RunCommand(command, parameters, new RunOptions());
```

Reply:
Fixed